spec/unit/application/lookup_spec.rb

```diff
@@ -78,7 +78,7 @@
   end
 
 
-  context 'when asked to explain' do
+  context 'when using a proper fixture' do
```

Comment:
No, as opposed to just "a configuration" that might be provided as ruby hashes or mocks.